app/responders/ropensci/reviewers_due_date_responder.rb

```diff
@@ -86,6 +90,18 @@ def due_date_in_days_from_now
       params[:due_date_days] || 21
     end
 
+    def respond_by_submission_type
+      generic_en = "#{reviewer} added to the reviewers list. Review due date is #{due_date}. Thanks #{reviewer} for accepting to review!"
+      replies = {
+        "standard" =>  "#{generic_en} Please refer to [our reviewer guide](https://devguide.ropensci.org/reviewerguide.html).",
+        "stats" => "#{generic_en} Please refer to [our reviewer guide](https://ropenscilabs.github.io/statistical-software-review-book/pkgreview.html).",
+        "estándar" => "#{generic_en} Please refer to [our reviewer guide](https://devguide.ropensci.org/reviewerguide.html).",
+      }
```

Comment:
Done, now adding replies for different submission types is just a matter of adding entries to this list.